docs-md/guides/live-reload.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Live Reload
+description: Use Live Reload to easily debug the web and native portions of an app on a device or simulator.
+url: /docs/guides/live-reload
+contributors:
+  - dotnetkow
+---
+
+# Live Reload
+
+Live Reload is useful for debugging both the web portion of an app as well as native functionality on device hardware or simulators. Rather than deploy a new native binary every time you make a code change, it reloads the browser (or WebView) when changes in the app are detected.
```

Comment:
```suggestion
Live Reload is useful for debugging both the web portion of an app as well as native functionality on device hardware or simulators. Rather than deploy a new native binary every time you make a code change, it reloads the browser (or Web View) when changes in the app are detected.
```

I think generally we prefer "Web View" vs "WebView".